OctoAwesome/OctoAwesome/Serialization/Entities/ComponentContainerDbContext.cs

```diff
@@ -36,11 +37,20 @@ public ComponentContainerDbContext(IDatabaseProvider databaseProvider, Guid univ
             entityDefinitionContext = new ComponentContainerDefinition<TComponent>.ComponentContainerDefinitionContext(database);
             componentsDbContext = new ComponentContainerComponentDbContext<TComponent>(databaseProvider, universe);
 
-            getComponentMethod = typeof(ComponentContainerComponentDbContext<TComponent>).GetMethod(nameof(ComponentContainerComponentDbContext<TComponent>.Get), new[] { typeof(TContainer) });
+            var getComponentMeth = typeof(ComponentContainerComponentDbContext<TComponent>).GetMethod(nameof(ComponentContainerComponentDbContext<TComponent>.Get), new[] { typeof(TContainer) });
```

Comment:
"getComponentMeth" ist aber ein eigenartiger Name, aber liegt vermutlich daran, dass "getComponentMethod" schon vergeben ist, aber da könnte man ruhig this und nicht this nutzen